### PR TITLE
chore: update documentation to new domain name

### PR DESCRIPTION
**Pull request by awentzel**
# Description
Updated all documentation references from fastdna.net to fast.design (new domain name).

## Motivation & context
Improved domain name better aligned to the future roadmap.

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: website/i18n/en.json [hunk 1]
```diff
@@ -37,3528 +37,6 @@
         "title": "Writing style guide",
         "sidebar_label": "Writing style guide"
       },
-      "en/packages/fast-animation/api/classes/_animate_index_.animate": {
-        "title": "en/packages/fast-animation/api/classes/_animate_index_.animate"
-      },
-      "en/packages/fast-animation/api/classes/_animatefrom_index_.default": {
```
Comment (nicholasrice) [inline website/i18n/en.json]: This looks generated - is there a reason we're tracking?

Comment (awentzel) [reply website/i18n/en.json]: interestingly it’s already tracked in ‘.gitignore’ so not sure whey it’s tracking.

Comment (janechu) [reply website/i18n/en.json]: I think you need to remove it from your gitignore, delete the file, then re-add it to the gitignore

Comment (awentzel) [reply website/i18n/en.json]: This file is already omitted from the commit.  Problem solved.

**Pull request merged**